python/featomic/tests/clebsch_gordan/equivariant_power_spectrum.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 import pytest
 from metatensor import Labels, TensorBlock, TensorMap
+from metatensor.operations import equal, join
```

Comment:
Please don't import from `metatensor.operations`. This could be written 

```
from metatensor import equal, join
```

Or even better

```suggestion
import metatensor as mts
```

and down the line `mts.equal`, `mts.join`

(like you would not do `from numpy import log` but instead use `np.log` where required)